### ISSUE TITLE
Feature Request: Blacklist

### ISSUE THREAD

**Issue by Bilibox** [JEFF-Media-GbR/Spigot-InvUnload#8]
Currently the plugin will search the whole inventory and dump all items into their respective chests, but this defeats the purpose of having extra pickaxes, shovels, etc

If a blacklist could be added so we can keep extra tools we normally use instead of it being dumped into chests it would be appreciated. 

Thank you

**Comment by mfnalex** [comment]
Good idea. Would you like this feature „per player“ or a global blacklist?

**Comment by Bilibox** [comment]
I was thinking about this for a while. If per player is possible that would be amazing. Maybe they can add to a blacklist through a command? Either or will do for me.

**Comment by mfnalex** [comment]
I will add this in the next update. :) I will also add support for the ChestSort API so that chests will be automatically sorted after unloading into them.
